_includes/svg-icons.html

```diff
@@ -12,3 +12,4 @@
 {% if site.footer-links.youtube %}<a href="https://youtube.com/{{ site.footer-links.youtube }}"><i class="svg-icon youtube"></i></a>{% endif %}
 {% if site.footer-links.googleplus %}<a href="https://plus.google.com/{{ site.footer-links.googleplus }}"><i class="svg-icon googleplus"></i></a>{% endif %}
 {% if site.footer-links.playconsole %}<a href="https://play.google.com/store/apps/dev?id={{ site.footer-links.playconsole }}"><i class="svg-icon playconsole"></i></a>{% endif %}
+{% if site.footer-links.mastodon %}<a rel="me" href="https://{{ site.footer-links.mastodon }}"><i class="svg-icon mastodon"></i></a>{% endif %}
```

Comment:
Note, in this link, I added`rel="me"`. This enables you to "verify" your account with mastodon. 

eg. in my mastodon profile, my site programmerdays.com, has a check mark in my profile. https://fosstodon.org/@danhan01010